src/includes/performance/traces-sampler-as-sampler/java.mdx

```diff
@@ -0,0 +1,27 @@
+```java
+import io.sentry.Sentry;
+
+Sentry.init(
+  options -> {
+    options.setDsn("___PUBLIC_DSN___");
+    // To set a uniform sample rate
+    options.setTracesSampleRate(1.0);
+    // Determine traces sample rate based on the sampling context
+    options.setTracesSampler(
+      context -> {
+        if (...) {
+          // These are important - take a big sample
+          return 0.5;
+        } else if (...) {
+          // These are less important or happen much more frequently - only take 1%
+          return 0.01;
+        } else if (...) {
+          // These aren't something worth tracking - drop all transactions like this
+          return 0;
```

Comment:
I think it would be nice to show some actual example here to help clarify the use case:

// something like this (That compiles)
```suggestion
        } else if (context.get("url") != null && context.get("url") == "/health")) {
          // The health check endpoint is just noise - drop all transactions
          return 0;
```

Reply:
Fixed